content/syllabuses/short-course-devops-foundational.md

```diff
@@ -0,0 +1,80 @@
+---
+title: DevOps Short Course – Foundational
+---
+
+Welcome to the DevOps Short Course, the foundational level.
+
+This course is to be completed in 6 weeks.
+
+Learners should spend 4 to 6 hours every week on self paced learning, including reading, working on tasks, quizzes etc.
+
+## General information
+
+### Group session structure
+We will have two 1h group sessions per week. They will mostly follow the structure below:
+
+#### Session 1
+- Review tasks and questions from last week
+- Workshop: the facilitator will present concepts, explain the content, ask questions, give answers, open a dialogue with the learners, demonstrate practical concepts
+- Task and homework: give the learners a task to do during the session, to be finished at home if needed and what tasks should be done at home
+
+#### Session 2
+- Workshop: same as above
+- Task and homework: same as above
+
+The week's tasks and quizzes must be sent by latest on Sunday.
+
+### How to submit answers to the conceptual questions
+Through out the course we will have conceptual questions requiring written answers. Follow the the instructions below to submit them:
+
+{{< contentlink path="project-submission-instructions/markdown-questions" >}}.
+
+## Course Content
+
+### Week 0: Introduction to Umuzi
+Here you will learn about tools and practices used through out an Umuzi short course.
+
+First, take the [Intro to Tilde for non-coder bootcampers]({{< ref "syllabuses/tilde-intro-non-tech-student" >}}).
+
+Then, learn how to use GitHub and Tilde.
+- {{< contentlink path="coding_aptitude_assessment/introduction_to_github" >}}
+- {{< contentlink path="projects/tilde/basic-repo-card-tutorial" flavour="" >}}
```

Comment:
I'm using the DS course as reference. There, we have three pieces of introduction content (below). Since it was there, I thought it would be the basic building blocks for a short course.

- `({{< ref "syllabuses/tilde-intro-non-tech-student" >}})`
- `{{< contentlink path="coding_aptitude_assessment/introduction_to_github" >}}`
- `{{< contentlink path="projects/tilde/basic-repo-card-tutorial" flavour="" >}}`

The Syllabus is quite big and I don't know most of what is there.

Since you are more aware about the Syllabus content than me, can you point me which introduction content should I give to the learners about Tilde, Git and GitHub?

Reply:
We basically have 2 intro syllabuses, one for techies and one for non-techies.

Looking over them, it looks like the one for techies "Intro to Tilde for tech bootcamps" is also inappropriate because it talks about success and failure at bootcamps. 

Maybe the way you set it up is best actually